inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -1900,33 +1881,33 @@ Form
 		RadioButtonGroup
 		{
 			Layout.columnSpan:		2
-			name:					"advanced_mu_transform"
+			name:					"advancedMuTransform"
 			title:					qsTr("Transform effect sizes")
-			id:						advanced_mu_transform
-			enabled:				measures_correlation.checked || measures_OR.checked
+			id:						advancedMuTransform
+			enabled:				measuresCorrelation.checked || measuresOR.checked
 
 			RadioButton
 			{
 				label:		qsTr("log(OR)")
 				value:		"log_OR"
```

Comment:
logOR?

Reply:
again, directly passed into the RoBMA function